javatests/arcs/core/entity/BUILD

```diff
@@ -35,7 +40,9 @@ arcs_kt_jvm_test_suite(
         "//java/arcs/flags:flags-kt",
         "//java/arcs/flags:unit_test_build_flags",
         "//java/arcs/flags/testing:testing-kt",
+        "//java/arcs/jvm/host",
         "//java/arcs/jvm/util/testutil",
+        "//javatests/arcs/core/host:testhost",
```

Comment:
It looks like you can revert all of these extra deps since they aren't needed by the tests in this build target.

Reply:
Ah yes, will do.

Reply:
Ah yes, will do.

Reply:
Not done?

Reply:
It was, but then I recombined the build targets (as per your suggested change for the force_enable flag on codegen). These targets are required for the type slicing test - it runs up a full arc to confirm that runtime handles are set up correctly.